ansible_collections/arista/avd/molecule/dhcp_provisioning/intended/configs/dhcpd.conf

```diff
@@ -17,6 +17,7 @@ host DC1-L2LEAF1A {
     option routers 192.168.200.5;
     option domain-name-servers 8.8.8.8, 192.168.200.5;
 }
+
```

Comment:
This change is making the CI fail

Reply:
weird. I did not change the file, but since I moved the directly I guess pre-commit changed all of them. I need to add exclusion to .conf then. Is dhcp really that sensitive?

Reply:
reverted the changes. They are caused by my updated libs, ansible and/or python3.11.